### ISSUE TITLE
MD header links a11y

### ISSUE THREAD

**Issue by missmatsuko** [missmatsuko/matsuko-ca#27]
Headers are auto-linked from gridsome-transform-remark.

By default, the links are `aria-hidden="true"`, but I'm wondering if they should be available to screen readers / AST?

If so, I need to figure out how to add accessible text to the links. I'm not sure if there's a way to grab the heading's text or ID for this purpose, though.
